media-gfx/arss/arss-0.2.3-r1.ebuild

```diff
@@ -14,11 +14,12 @@ SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.gz"
 LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE=""
 
 DEPEND="sci-libs/fftw:3.0="
 RDEPEND="${DEPEND}"
 
-S=${WORKDIR}/${MY_P}/src
+S="${WORKDIR}/${MY_P}/src"
+
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
```

Comment:
```suggestion
PATCHES=( "${FILESDIR}/${P}-fno-common.patch" )
```

Reply:
nope, @jsmolic has the canonical style correct

Reply:
Shouldn't all variables be quoted?

Reply:
This is a style preference, both ways are correct. The important part is that `FILESDIR` is quoted since *sigh* it can contain spaces.

Reply:
Ah makes sense, thanks for the info